src/components/dragdropLayout.tsx

```diff
@@ -32,11 +32,6 @@ function DragdropLayout(props: CustomLayoutProps) {
   return (
```

Comment:
also remove obsolete hook L30

Reply:
Done

Reply:
Hey @oyo ,
Could you please check these changes?
Thanks!